src/Providers/Transforms/EmailConfiguration/EmailConfigurationTransformDataProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+
+namespace form_builder.Providers.Transforms.EmailConfiguration
+{
+    public class EmailConfigurationTransformDataProvider : IEmailConfigurationTransformDataProvider
+    {
+        public async Task<T> Get<T>()
+        {
+            return JsonConvert.DeserializeObject<T>(
+                await System.IO.File.ReadAllTextAsync($@".\DSL\email-config\emailconfiguration.json"));
+        }
```

Comment:
Suggestion:
As an out of the box solution you could use System.Text.Json to Deserialse Async... eg.
using FileStream openStream = File.OpenRead($@".\DSL\email-config\emailconfiguration.json");
            return await System.Text.Json.JsonSerializer.DeserializeAsync<T>(openStream);